modules/ROOT/pages/database-administration/standard-databases/seed-from-uri.adoc

```diff
@@ -133,38 +178,73 @@ CREATE DATABASE foo OPTIONS { seedURI: 'azb://myStorageAccount/myContainer/myBac
 
 Starting from Neo4j 2025.01, the `CloudSeedProvider` supports seeding up to a specific date or transaction ID using the `seedRestoreUntil` option.
 
-[role=label--new-2025.01]
+
 Seed up to a specific date::
 
 To seed up to a specific date, provide the differential backup containing the data up to that date.
 +
-[source,cypher]
+[.tabbed-example]
+=====
+[role=include-with-Cypher-5]
+======
+[source,cypher5]
+----
+CREATE DATABASE foo OPTIONS {
+    existingData: 'use',
+    seedURI: 's3://myBucket/myBackup.backup',
+    seedRestoreUntil: datetime('2019-06-01T18:40:32.142+0100')
+}
+----
+======
+[role=include-with-Cypher-25]
+======
+
+[source,cypher25]
 ----
 CREATE DATABASE foo OPTIONS {
     seedURI: 's3://myBucket/myBackup.backup',
     seedRestoreUntil: datetime('2019-06-01T18:40:32.142+0100')
 }
 ----
+======
+=====
 +
 This will seed the database with transactions committed before the provided timestamp.
 
-[role=label--new-2025.01]
+
 Seed up to a specific transaction ID::
 
 To seed up to a specific transaction ID, provide the differential backup containing the data up to that transaction ID.
 +
-[source,cypher]
+[.tabbed-example]
+=====
+[role=include-with-Cypher-5]
+======
+[source,cypher5]
+----
+CREATE DATABASE foo OPTIONS {
+    existingData: 'use',
+    seedURI: 's3://myBucket/myBackup.backup',
+    seedRestoreUntil: 123
+}
+----
+======
+[role=include-with-Cypher-25]
+======
+[source,cypher25]
 ----
 CREATE DATABASE foo OPTIONS {
     seedURI: 's3://myBucket/myBackup.backup',
     seedRestoreUntil: 123
 }
 ----
+======
+=====
 +
 This will seed the database with transactions up to (but not including) transaction 123.
 
 
-[role=label--deprecated]
+[role=label--deprecated label--cypher-5]
```

Comment:
see line 251 which is commented out

Reply:
You could remove the label on line 251 then.